### ISSUE TITLE
JAWS 2019 focus is not going to modal dialogues control and stays on parent screen.

### ISSUE THREAD

**Issue by awalia13** [FreedomScientific/standards-support#160]
I am using JAWS 2019.1812.49 ILM version and in this version JAWS virtual cursor focus is not going to Modal dialogues controls automatically when accessing dialogues using up/down arrow keys.

>Go to URL: https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html
>Turn on JAWS and navigate the url using JAWS virtual cursor mode (using up/down arrow keys)
> Click on "Add Delivery Address Button". It will open dialogue box.
> Click on "Verify Address" button and it will open "Verification Results" dialogue.
> Click on "accepting an alternative form" using down arrow key. It will open dialogue box "End of Road".
> Now notice JAWS behaviour.JAWS focus remains on the Parent Screen (Verification Result) button "accepting and alternative form" and when user navigate the newly opened dialogue using up/down arrow keys, it reads the controls of parent screen instead of reading the controls of popup (End of Road)

### Expected result

JAWS focus sholuld land on the most interactive element of dialogue box and user should be able to navigate to dialogue box using up/down arrow keys.

### Actual result

JAWS focus remains on the Parent screen instead and user is not able to access dialogue box using up/down arrow keys. 

Note: This is working fine with JAWS 2018.1811.12 ILM

[JAWS Modal Dialogue.zip](https://github.com/FreedomScientific/VFO-standards-support/files/2767129/JAWS.Modal.Dialogue.zip)

 version.

#### JAWS version and build number
JAWS 2019.1812.49 ILM

#### Operating System and version
Windows 10 Version 1809(OS Build 17763.253)

#### Browser and version:
Chrome Browser and Version 71.0.3578.98 (Official Build) (64-bit)

**Comment by scottaohara** [comment]
This issue may be due to the modal dialog stacking.  JAWS does seem to be getting confused with where focus should go / goes in and out of application mode when navigating by virtual cursor when the second stacked dialog is loaded.

JAWS 2018 is not without issues here either. The virtual cursor does work as expected with this same demo, however the content of the base document becomes accessible again.

Both versions of JAWS, tested with Firefox and Chrome, work as expected when only the initial dialog has been opened.

**Comment by awalia13** [comment]
Hi Scott, Thanks for the quick reply. In our application we have the same scenario when multiple dialogue box open one after other and on the top dialogue box JAWS virtual cursor focus is not landing due to this user is not able to access the dialogue box. But this working fine with JAWS 2018.

Also you have added JAWS bug and Regression tag in the bug itself. Just confirming this is JAWS regression bugs???

Thanks for your help in advance.

**Comment by scottaohara** [comment]
hi @awalia13 
from initial testing it appears to be a regression since this same behavior didn't occur in 2018.

Once we can fully verify and get more info, status/labels may change.  Any updates we have will be posted here :)

**Comment by sachintiwarimg** [comment]
>

**Comment by kurtjwolf** [comment]
Internally filed as bug #107769 on 3/22/2019.

**Comment by themadhaggis** [comment]
Do we have a timescale on when this issue might be resolved?

Thanks
